test/SimpleClientTests/SchemaParsingSpec.hs

```diff
@@ -0,0 +1,29 @@
+-- This module contains unit tests testing that the client can correctly parse
+-- specific JSON files according to the given schema.
+{-# OPTIONS_GHC -Wno-deprecations #-}
+module SimpleClientTests.SchemaParsingSpec where
+
+import qualified Data.ByteString as BS
+import qualified Data.Aeson as AE
+
+import Concordium.Client.Types.Contract.Parameter
+import Concordium.Client.Types.Contract.Schema
+
+import Test.Hspec
+import Test.HUnit
+
+-- Test that parsing a parameter that contains a set, a 32-byte array and an
```

Comment:
Shouldn't it be uleb128/ileb128 instead of `set`?

Reply:
As we discussed in the office, it is a set.

I have however added a ULEB128 to the test now as well.